site/docs/latest/security/tls.md

```diff
@@ -152,7 +152,7 @@ For more information on Pulsar client authentication using TLS, see the followin
 You'll need to add the following authentication parameters to that file to use TLS with Pulsar's CLI tools:
 
 ```properties
-serviceUrl=https://broker.example.com:8443/
+webServiceUrl=https://broker.example.com:8443/
```

Comment:
Add brokerServiceUrl=pulsar+ssl://broker.example.com:6651/ also. pulsar-client needs it.